source/manual/covid-19-services.html.md

```diff
@@ -26,7 +26,7 @@ services such as medical equipment, hotel rooms or childcare.
 
 ## Get coronavirus support as an extremely vulnerable person
 
-This service allows citizens identified as vulnerable by the NHS to
+This service allows the public identified as vulnerable by the NHS to
```

Comment:
maybe 'people' is better here?

Reply:
Or maybe "individuals"? 🤔

Reply:
I like "people" too. If not, maybe "members of the public" reads more naturally here.